docs/existing-aws-infrastructure.md

```diff
@@ -0,0 +1,94 @@
+# Consuming Existing AWS Infrastructure
+
+Normally, Cluster API will create infrastructure on AWS when standing up a new workload cluster. However, it is possible to have Cluster API re-use existing AWS infrastructure instead of creating its own infrastructure. Follow the instructions below to configure Cluster API to consume existing AWS infrastructure.
+
+## Prerequisites
+
+In order to have Cluster API consume existing AWS infrastructure, you will need to have already created the following resources:
+
+* A VPC
+* One or more private subnets (subnets that do not automatically assign a public IP address to EC2 instances)
+* A public subnet in the same Availability Zone (AZ) for each private subnet (this is required for NAT gateways to function properly)
+* A NAT gateway for each private subnet, along with associated Elastic IP addresses
+* An Internet gateway for all public subnets
+* Route table associations that provide connectivity to the Internet through a NAT gateway (for private subnets) or the Internet gateway (for public subnets)
+
+You will need the following information, which can be obtained either via the AWS Management Console or using the AWS CLI:
+
+1. The ID of the VPC
+2. The IDs of the subnets in the VPC that you want Cluster API to use
+
+Note that there is no need to create an Elastic Load Balancer (ELB), security groups, or EC2 instances; Cluster API will take care of these items.
+
+## Tagging AWS Resources
```

Comment:
I wouldn't expect the tags to be required if specifying vpc ids and subnet ids, but I'll try to verify.

Reply:
Agreed – they're not required, and in fact not a good idea. The way CAPA knows whether it can modify the VPC/subnets/route tables/nat gateways/IGWs is by whether the VPC has the "owned" tag. If we mention the tags at all in this page, I would suggest saying something to the effect of "setting this tag will tell CAPA to take ownership over and modify the existing resources" and leaving it at that.

Reply:
References to CAPI-specific tags removed via 38ce860.

Reply:
Resolved via 38ce860, where I removed references to CAPI-specific tags being required.